client/src/utils/css-help.test.ts

```diff
@@ -0,0 +1,97 @@
+/* global describe it expect */
+import { cssString } from './__fixtures/curriculum-helpers-css';
+import CSSHelp from './css-help';
+
+describe('css-help', () => {
+  document.body.innerHTML = '';
+  document.head.innerHTML = '';
```

Comment:
```suggestion
```
This is handled by `afterEach`

Reply:
But, it is not handled by beforeEach. I thought its purpose was to clear the document of previous tests (other `.test.js` files)?

Reply:
Right, so the flow is this:

1. beforeEach: populate the document with the fixtures
2. run test
3. afterEach: clear out the document of all content (fixtures + whatever changes the test made)
4. repeat for the next test


The document starts empty (at least I assume it does), so you shouldn't need to empty it before step 1.